liota/device_comms/cipethernetip_device_comms.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think package name should be cip_ethernet_ip_device_comms

Reply:
ok